### ISSUE TITLE
Prepare code for 0.15

### ISSUE THREAD

**Issue by mbevc1** [terraform-aws-modules/terraform-aws-iam#146]
0.15 was released and in order to use modules here we'd need to remove deprecated syntax like: https://github.com/terraform-aws-modules/terraform-aws-iam/blob/master/modules/iam-assumable-role/main.tf#L2

**Comment by antonbabenko** [comment]
Sure, please make a PR :)

**Comment by andormarkus** [comment]
Hi @mbevc1 

outputs are causing errors as well.


```
│ Error: Output refers to sensitive values
│ 
│   on .terraform/modules/this/modules/iam-user/outputs.tf line 43:
│   43: output "this_iam_access_key_secret" {
│ 
│ Expressions used in outputs can only refer to sensitive values if the
│ sensitive attribute is true.
╵
╷
│ Error: Output refers to sensitive values
│ 
│   on .terraform/modules/this/modules/iam-user/outputs.tf line 58:
│   58: output "this_iam_access_key_ses_smtp_password_v4" {
│ 
│ Expressions used in outputs can only refer to sensitive values if the
│ sensitive attribute is true.
╵

```

**Comment by mbevc1** [comment]
I'll try to open a PR in next few days - time permitting :)

**Comment by andormarkus** [comment]
Hi @antonbabenko 

**Legacy functions**
Changing the `list` to `tolist` function will not break the module because that function is available from v0.12.0 and the module is supporting terraform from v0.12.6

**Sensitive Output Values**
The sensitive output are causing backwards compatibility issues. It was introduced in v0.14.0 and in v0.15.0 they made it more sensitive  [[source](https://www.terraform.io/upgrade-guides/0-15.html#sensitive-output-values)].

We got two options:
- ` sensitive = true` in output  - need v0.14.0
- `nonsensitive` function - needs v0.15.0

To solving the sensitive output values problem we need to raise the minimum supported version of the module. [Terraform suggest](https://www.terraform.io/upgrade-guides/0-15.html#sensitive-output-values) to raise the minimum supported version to v0.15.0
We need guidance in this manner.

**Comment by antonbabenko** [comment]
Setting `sensitive = true` on outputs seems to be the best option that works for all supported versions (starting from Terraform 0.12.6).

Fixed in #148.

[v3.15.0](https://github.com/terraform-aws-modules/terraform-aws-iam/releases/v3.15.0) has been just released.

**Comment by mbevc1** [comment]
Beat me to it. Thanks @antonbabenko :partying_face:
